android/vision/README.md

```diff
@@ -0,0 +1,45 @@
+
```

Comment:
1. Please add step-to-step instructions on how the program can be run so that anyone could self-help run the program by reading the instructions.
2. Please also add instructions on how the verify and test the program.

Reply:
The instructions are in the README file but I'll update it to make it more clear.